ICSharpCode.Decompiler/CSharp/CallBuilder.cs

```diff
@@ -1438,7 +1438,7 @@ private ExpressionWithResolveResult HandleImplicitConversion(IMethod method, Tra
 			var conversions = CSharpConversions.Get(expressionBuilder.compilation);
 			IType targetType = method.ReturnType;
 			var conv = conversions.ImplicitConversion(argument.Type, targetType);
-			if (!(conv.IsUserDefined && conv.IsValid && conv.Method.Equals(method)))
+			if (!(conv.IsUserDefined && conv.IsValid && conv.Method.Equals(method, NormalizeTypeVisitor.IgnoreNullabilityAndTuples)))
```

Comment:
Why only ignore nullability and tuples?
I think normally when comparing if a method we found in the type system matches a method present in IL, we use `NormalizeTypeVisitor.TypeErasure` which also handles dynamic/object and IntPtr/nint.